File: #5 The Hashtag Generator/main.py
def generate_hashtag(s):
    if not s or s.strip() == "":
        return False

    words = s.strip().split()
    capitalized_words = [word.upper() for word in words]

    result = "#" + "".join(capitalized_words)
    return result if len(result) <= 140 else False

File: #5 The Hashtag Generator/main.js
function generateHashtag(str) {
  if (!str || str.trim() === "") return false;

  const words = str.trim().split(/\s+/);

  const capitalizedWords = words.map(
    word => word.charAt(0).toUpperCase() + word.slice(1).toLowerCase()
  );

  const result = "#" + capitalizedWords.join("");

  return result.length > 140 ? false : result;
}
